catalog-rest-service/src/main/resources/ui/webpack.config.dev.js

```diff
@@ -47,7 +47,7 @@ module.exports = {
       // .js and .jsx files to be handled by babel-loader
       {
         test: /\.(js|jsx)$/,
-        exclude: /node_modules/,
+        // exclude: /node_modules/,
```

Comment:
Why we need to include this?

Reply:
for react-slick webpack is throwing error while parsing the files, after including node_module its running properly

Reply:
We don't need to include node_modules for js | jsx files

Reply:
updated